docs/en/ingest-management/commands.asciidoc

```diff
@@ -31,31 +31,88 @@ NOTE: You might need to log in as a root user to run these commands.
 [[elastic-agent-enroll-command]]
 == elastic-agent enroll
 
-Enroll the {elastic-agent} in {fleet}.
+Enroll the {agent} in {fleet}.
+
+Use this command to enroll the {agent} in {fleet} without installing the
+agent as a service. You might want to do this if you installed
+the {agent} from a DEB or RPM package and plan to use systemd commands to
+start and manage the service. This command is also useful for testing
+{agent} prior to installing it.
 
-We recommend that you run this command as the root user because some
+TIP: To enroll an {agent} _and_ install it as a service, use the
+<<elastic-agent-install-command,`install` command>> instead.
+
+We recommend that you run the `enroll` command as the root user because some
 integrations require root privileges to collect sensitive data. This command
 overwrites the `elastic-agent.yml` file in the agent directory.
 
+This command includes optional flags to set up <<fleet-server,{fleet-server}>>.
+
+IMPORTANT: This command enrolls the {agent} in {fleet}; it does not start the
+agent. To start the agent, either <<start-elastic-agent,start the service>>, if
+one exists, or use the <<elastic-agent-run-command,`run` command>> to start the
+agent from a terminal.
+
+
```

Comment:
```suggestion
The most common usage is to use the 'install' command described below, which entails management of the {agent} service.  The 'enroll' command can be used after installing the {agent} to modify the settings {agent} runs with.
```

Reply:
The first sentence seems a little redundant with what I've already said. I'll try to make that statement clearer and mention the point about using enroll after installation.